dapps/dapps_dev.json

```diff
@@ -922,6 +922,14 @@
         "utilities"
       ],
       "desktopOnly": true
+    },
+    {
+      "name": "Incognitee Privacy Sidechains Dapp",
+      "url": "https://app.incognitee.io/",
```

Comment:
```suggestion
      "url": "https://app.incognitee.io/?forceLive=true",
```

Reply:
yeah, we can remove that tomorrow, but it won't hurt anyway